sql/005_create_business_posting_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `business_posting` (
+  `business_name` varchar(45) NOT NULL,
```

Comment:
`business_name` makes for a bad primary key. Primary key has to be unique. It is totally fine to have two business /posting for two separate 7/11 stores.

That is going to create a conflict.